Sources/SwifterSwift/Dispatch/DispatchQueueExtensions.swift

```diff
@@ -56,6 +56,21 @@ public extension DispatchQueue {
         asyncAfter(deadline: .now() + delay, qos: qos, flags: flags, execute: work)
     }
 
+    func debounce(millisecondsDelay: Int, action: @escaping () -> Void) -> () -> Void {
```

Comment:
Why milliseconds, not a `DispatchTimeInterval`?

Reply:
It's how the original function was written. More than happy to make it a `TimeInterval`, as it's more standard.

Reply:
I think that it would be nicer :)